styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -107,9 +106,9 @@
   private final WorkflowExecutionGate gate;
 
   public Scheduler(Time time, TimeoutConfig ttls, StateManager stateManager,
-      WorkflowCache workflowCache, Storage storage,
-      WorkflowResourceDecorator resourceDecorator,
-      Stats stats, RateLimiter dequeueRateLimiter, WorkflowExecutionGate gate) {
+                   WorkflowCache workflowCache, Storage storage,
+                   WorkflowResourceDecorator resourceDecorator,
+                   Stats stats, RateLimiter dequeueRateLimiter, WorkflowExecutionGate gate) {
     this.time = Objects.requireNonNull(time);
     this.ttls = Objects.requireNonNull(ttls);
     this.stateManager = Objects.requireNonNull(stateManager);
```

Comment:
Please help fix this null check. :)

Reply:
What does it mean? :)

Reply:
I don't see that line with missing check :/

Reply:
Please disregard this comment. I actually meant `QueuedStateManager` constructor.